typescript/demo/function-call-stream.ts

```diff
@@ -217,18 +209,20 @@ async function functionCallingWithAIConfig() {
       return;
     }
 
-    const message = output.data as ChatCompletionMessageParam;
+    const rawResponse = output.metadata?.rawResponse;
+    const function_call = rawResponse?.function_call;
+    console.log("function_call=", function_call);
```

Comment:
nit, can remove

Reply:
I wanted to keep it because we have other `console.logs()` earlier and it shows more clearly in the demo how function_call is being used. Ex: https://github.com/lastmile-ai/aiconfig/blob/main/typescript/demo/function-call-stream.ts#L176